src/Compilers/Core/MSBuildTaskTests/CscTests.cs

```diff
@@ -334,21 +334,30 @@ public void RefOnly()
         }
 
         [Fact]
-        public void NullableReferenceTypes_True()
+        public void NullableReferenceTypes_Enabled()
```

Comment:
Nit: could make these tests a `Theory`, and then have one test with the 3 options.